builder/oracle/classic/step_list_images.go

```diff
@@ -98,6 +97,29 @@ func (s *stepListImages) Run(_ context.Context, state multistep.StateBag) multis
 	return multistep.ActionContinue
 }
 
+func (s *stepListImages) listForWinRM(state multistep.StateBag) multistep.StepAction {
```

Comment:
imho this is a premature optimization

Reply:
you're so fast you somehow travel back in time

Reply:
lulz